tests/aws/services/cloudwatch/test_cloudwatch.validation.json

```diff
@@ -20,6 +20,9 @@
   "tests/aws/services/cloudwatch/test_cloudwatch.py::TestCloudwatch::test_enable_disable_alarm_actions": {
     "last_validated_date": "2023-09-12T10:00:45+00:00"
   },
+  "tests/aws/services/cloudwatch/test_cloudwatch.py::TestCloudwatch::test_get_metric_with_null_dimensions": {
+    "last_validated_date": "2024-01-09T20:13:11+00:00"
+  },
```

Comment:
for some reason the validation date for `test_get_metric_with_no_results` is missing - did you forget to add it to the commit?

Reply:
My guess for that was that the snapshot was simply created before the new validation system was on master. Which seemed fine to me, since we have a bunch of snapshots like that right now. The other test is in the validation file here because due to a hiccup yesterday it had to be re-generated

Reply:
@baermat: Actually I thought we should have validation files for all existing snapshots, as this was added with https://github.com/localstack/localstack/pull/9928. Maybe some slipped through because they were merged in PRs in between.

Reply:
ah my bad, then disregard please, and yes that should be re-generated then